pycbc/waveform/bank.py

```diff
@@ -381,10 +378,12 @@ def write_to_hdf(self, filename, start_index=None, stop_index=None,
         write_tbl = self.table[start_index:stop_index]
         for p in parameters:
             f[p] = write_tbl[p]
-        if self.compressed_waveforms is not None:
+        if self.check_compressed_waveforms:
```

Comment:
Add a `write_compressed_waveforms=True` keyword argument to `write_to_hdf`, then have this if statement check for `if write_compressed_waveforms and self.has_compressed_waveforms`. It occurs to me that someone might want to write out a copy of the bank that just has the template parameters.